pages/people/[id]/details.tsx

```diff
@@ -4,17 +4,40 @@ import { GetServerSideProps } from 'next';
 import { Resident } from 'types';
 import PersonDetails from 'components/PersonView/PersonDetails';
 import { isAuthorised } from '../../../utils/auth';
+import { useRouter } from 'next/router';
+import { useState, useEffect } from 'react';
+import ConfirmationBanner from 'components/ConfirmationBanner/ConfirmationBanner';
+import { ConditionalFeature } from 'lib/feature-flags/feature-flags';
 
 interface Props {
   person: Resident;
 }
 
-const PersonAllocationsPage = ({ person }: Props): React.ReactElement => (
-  <Layout person={person}>
-    <PersonDetails person={person} />
-  </Layout>
-);
+const PersonAllocationsPage = ({ person }: Props): React.ReactElement => {
+  const [showConfirmation, setShowConfirmation] = useState(false);
```

Comment:
Since this is specific for case status, maybe something like `showAddedCaseStatusConfirmation` is clearer?

Reply:
Renamed the variable